lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -19,7 +19,12 @@ const log = require('lighthouse-logger');
 const {runSmokehouse} = require('../smokehouse.js');
 const {server, serverForOffline} = require('../../fixtures/static-server.js');
 
-const coreTestDefnsPath = require.resolve('../test-definitions/core-tests.js');
+let coreTestDefnsPath = '';
```

Comment:
does `path.join(__dirname, '../test-definitions/core-tests.js')` work instead? this try/catch feels a bit awkward

Reply:
yes but i was opting to not change the string value.